brainiak/funcalign/fastsrm.py

```diff
@@ -926,7 +926,9 @@ def _compute_subject_basis(corr_mat):
     or shape=[n_components, n_supervoxels]
         basis of subject or reduced_basis of subject
     """
-    U, _, V = scipy.linalg.svd(corr_mat, full_matrices=False)
+    perturbation = np.zeros(corr_mat.shape)
```

Comment:
It would be good to add a comment as to why you needed to perturb the correlation matrix before running the SVD

Reply:
This is for consistency reasons with respect to previous implementation of SRM. Running FastSRM with no atlas is now equivalent to running DetSRM which is a good thing (I added a comment in the code)